rolling_update.yml

```diff
@@ -59,7 +63,7 @@
       until: result.rc == 0
       retries: 5
       delay: 10
-      delegate_to: 127.0.0.1
+      delegate_to: "{{ groups.mons[0] }}"
```

Comment:
My approach takes the last non-self monitor. It seems I should take the first non-self one. Does it?